client/src/components/Spreadsheet.jsx

```diff
@@ -0,0 +1,109 @@
+import 'tui-grid/dist/tui-grid.css';
+import Grid from '@toast-ui/react-grid';
+import React, { useState, useEffect, useCallback, useRef } from 'react';
+import {Modal,FormGroup } from 'react-bootstrap'
+import Button from './Button'
+import Input from "../components/Input";
+import CancelButton from "./CancelButton";
+import ModalUpdate from './ModalUpdate';
+const { REACT_APP_PROXY } = process.env;
+
+export default function SpreadsheetToastGrid(){
+  const [dataArray, setDataArray] = useState([])
+  const [show, setShow] = useState(false);
+  const [selectedCoop, setSelectedCoop] = useState([])
+  const handleClose = () => {
+    setShow(false);
+    setSelectedCoop([])
```

Comment:
Minor thing, but I notice sometimes you use semicolons at the end of lines and other times not, any specific reason?

Reply:
Thanks for calling that out - that's sloppy on my part. I typically use a vscode extension to auto-add semicolons at the end of lines, but it doesn't seem to be working.